src/styles/blocks/product.css

```diff
@@ -0,0 +1,84 @@
+html {
+  font-family: 'Roboto', sans-serif;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
```

Comment:
we don't need to add left/right paddings for each card element, just add paddings to card for all sides